src/planner/macos.rs

```diff
@@ -110,9 +125,13 @@ impl Planner for Macos {
             },
         };
 
-        let encrypt = match self.encrypt {
-            Some(choice) => choice,
-            None => {
+        // The encrypt variable isn't used in the enterprise edition since we have our own plan step for it,
+        // however this match accounts for enterprise edition so the receipt indicates encrypt: true.
+        // This is a goofy thing to do, but it is in an attempt to make a more globally coherent plan / receipt.
```

Comment:
nit:

Something I Just realized is: instead, we could modify line 261 to be:

```rust
map.insert("volume_encrypt".into(), serde_json::to_value(encrypt || self.enterprise_edition)?);
```

and then we don't need this goofy match and we'll keep `encrypt = true` in the plan's JSON.

Reply:
That'd be lying